.config/tsaoptions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
+  "codebaseName": "Orleans",
+  "instanceUrl": "https://devdiv.visualstudio.com/",
```

Comment:
Since we are still building on our own devops instance, should we change this value?

Reply:
I don't know because the documentation of this file format is sorely lacking. I suspect this is the location where `areaPath` lives and where TSA bugs will be filed. Suggest we probably want to keep @danmoseley's bug in the same instance. @dcwhittaker do you know more❔

Reply:
I believe this can be resolved since this instance is definitely where the area path lives.